_docs/releasing_version.md

```diff
@@ -60,8 +92,23 @@ regenerate it.
 
 ## Doing the release
 
+*Please note that in order to execute this rake task you must be in the __Collaborators__ group on GitHub for the module in question.*
+
+*Please also note that the task requires a configured gpg or ssh key in your local git settings to sign the git tag*
+
+This step must be done by a voxpupuli maintainer!
+
 This has to be done on the __*upstream*__ repo itself.
 
+If you are switching from contributor to maintainer, you need to rename the remotes to reflect your status:
+
+```bash
```

Comment:
to prevent this I recommend cloning the original repo first and then adding the personal fork as additonal remote. That's also the default setup created by msync.